### ISSUE TITLE
Separating of flags

### ISSUE THREAD

**Issue by sebastian-c** [SWS-Methodology/faoswsTrade#8]
See also SWS-868 on JIRA. The flags for trade need to be separated into observations status and method flags.

**Comment by sebastian-c** [comment]
It looks like this is accomplished in 2e5aa77489e578cd26f7c10ebab53aa3ab890d29, but we're not quite ready yet. I'm proposing to revert this change and then reapply it when we're ready to implement it.

**Comment by sebastian-c** [comment]
Ok, in regards to this issue, the new tables with M49 also have the new flag setup. I reapplied the modification, but I get this error:

```
> esdata <- esdata %>%
+ left_join(fclunits, by = "fcl")
Error: cannot join on columns 'fcl' x 'fcl': index out of bounds 
> str(esdata)
Classes ‘tbl_df’, ‘tbl’ and 'data.frame':   0 obs. of  9 variables:
 $ reporter: num 
 $ partner : num 
 $ hs      : chr 
 $ flow    : int 
 $ year    : chr 
 $ value   : num 
 $ weight  : num 
 $ qty     : num 
 $ hs6     : chr 
```

I think I may have set up faulty limits, though

**Comment by sebastian-c** [comment]
We won't be able to confirm this issue as fixed until we fix #17
